lib/process/import.go

```diff
@@ -112,12 +112,12 @@ func (i *importer) Close() error {
 
 // getMasterTeleportConfig extracts configuration from teleport package
 func (i *importer) getMasterTeleportConfig(clusterName string) (*telecfg.FileConfig, error) {
-	configPackage, err := pack.FindLatestPackageCustom(pack.FindLatestPackageRequest{
-		Packages:   i.packages,
-		Repository: clusterName,
-		Match:      matchTeleportConfigPackage(*defaults.TeleportVersion),
-	})
+	configPackage, err := i.findLatestTeleportConfigPackage(clusterName, *defaults.TeleportVersion)
```

Comment:
To ease debugging, do you know if we eventually log the package that was selected somewhere? If not, I'd log it here.

Reply:
Yes, it is logged at the caller's site.